tests/unit_tests/test_array_find.py

```diff
@@ -0,0 +1,46 @@
+import json
+
+import pytest
+
+from tests.utils import load_process_code, run_process, run_input_validation
+
+
+@pytest.fixture
+def array_find_process_code():
+    return load_process_code("array_find")
+
+
+@pytest.mark.parametrize(
+    "example_input,expected_output",
+    [
+        ({"data": [1, 2, 3], "value": 2}, 1),
+        ({"data": ["A", "B", "C"], "value": "b"}, None),
+        ({"data": [1, 2, 3], "value": "2"}, None),
+        ({"data": [1, None, 2, None], "value": None}, None), # example in docs shows "1" as a return, but description states "None"
```

Comment:
The comment can be removed now, since I see that they said in the issue https://github.com/Open-EO/openeo-processes/issues/348 that it was fixed. Same goes for the comment in the `test_array_contains.py`